python/util/templates.py

```diff
@@ -63,15 +66,19 @@ def generate_react_components(self):
         """ Generate javascript to render every component which has been marked
             used in this ComponentsDatabase, if any.
         """
+        view_embeds = public_views_embed.get(flask.request.endpoint, {})
         script = """
 document.addEventListener("DOMContentLoaded", function () {
     'use strict';
-    var components;
-"""
+
+    var """
+        for key, value in view_embeds.items():
+            script += str(key) + " = " + json.dumps(value) + ","
```

Comment:
it looks like your keys are going to be like `domain_embed`, `products_embed`, which are not nice javascript mixedCase variables.

Reply:
Not a massive issue, I could update the keys if needed.